Makefile

```diff
@@ -73,11 +74,14 @@ vcctl: init
 
 image_bins: init
 	GO111MODULE=off go get github.com/mitchellh/gox
-	CGO_ENABLED=0 $(GOBIN)/gox -osarch=${REL_OSARCH} -ldflags ${LD_FLAGS} -output ${BIN_DIR}/${REL_OSARCH}/vcctl ./cmd/cli
-	for name in controller-manager scheduler webhook-manager; do\
-		CGO_ENABLED=0 $(GOBIN)/gox -osarch=${REL_OSARCH} -ldflags ${LD_FLAGS} -output ${BIN_DIR}/${REL_OSARCH}/vc-$$name ./cmd/$$name; \
+	CC=${CC} CGO_ENABLED=0 $(GOBIN)/gox -osarch=${REL_OSARCH} -ldflags ${LD_FLAGS} -output ${BIN_DIR}/${REL_OSARCH}/vcctl ./cmd/cli
+	for name in controller-manager webhook-manager; do\
+		CC=${CC} CGO_ENABLED=0 $(GOBIN)/gox -osarch=${REL_OSARCH} -ldflags ${LD_FLAGS} -output ${BIN_DIR}/${REL_OSARCH}/vc-$$name ./cmd/$$name; \
 	done
 
+	# scheduler need cgo enabled to support plugin
+	CC=${CC} CGO_ENABLED=1 $(GOBIN)/gox -osarch=${REL_OSARCH} -ldflags ${LD_FLAGS} -output ${BIN_DIR}/${REL_OSARCH}/vc-scheduler ./cmd/scheduler;
```

Comment:
hi @zen-xu , it will cause some problem using `gcc` with CGO enabled, l think it will be better to specify the `musl-gcc` in Makefile.

FYI: l just ran `make images` directly to build images and there's something wrong.

Reply:
what's the wrong message?

Reply:
`standard_init_linux.go:211: exec user process caused "no such file or directory"`

It should be a cross compilation problem with CGO enabled

Reply:
yes, I know the reason

Reply:
Since CI use ubuntu as default image, so I think scheduler should also use `ubuntu`.

Reply:
`Makefile` is not only used by CI, but users on different platforms may use it. It should not only support a specific platform

Reply:
If you use `gcc` build the scheduler, you should promise your container has gnu-libc
If you use `musl-gcc` build the scheduler, you should promise your container has musl-libc

Reply:
But this is a strong assumption. The compilation method needs to be changed according to the environment in which the image is deployed. E.g. I am currently on ubuntu, and I just want to compile vc-scheduler image(with alpine as basic image) through `make images`,  but it doesn't work and you don't even tell me to use `musl-gcc`

Reply:
I write a [design doc](https://github.com/zen-xu/volcano/blob/custom-plugin-design/docs/design/custom-plugin.md), But has not been merged yet.

Reply:
okay, now l got it.. but it is still confusing if someone doesn't use the `Custom Plugin` need to know that)